android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/ConsentManager.java

```diff
@@ -157,38 +195,19 @@ public Completable giveConsent(@NonNull String subpopulationGuid, @NonNull Strin
                 .imageMimeType(imageMimeType)
                 .scope(sharingScope);
 
-        return storeConsentSignatureLocally(subpopulationGuid, consentSignature)
-                .andThen(
-                        RxUtils.toBodySingle(forConsentedUsersApi.
-                                createConsentSignature(subpopulationGuid, consentSignature))
-                                .compose(safeGetNewSessionOnSuccess())
-                                .toCompletable()
-                                .onErrorComplete(e -> {
-                                    LOG.info("Couldn't upload consent to Bridge, " +
-                                            "subpopulationGuid: " + subpopulationGuid, e);
-                                    return true;
-                                }));
-    }
-
-    private Completable storeConsentSignatureLocally(@NonNull String subpopulationGuid,
-                                                     @NonNull ConsentSignature consentSignature) {
-        LOG.debug("Saving consent locally, subpopulationGuid: " + subpopulationGuid);
+        storeConsentSignatureLocally(subpopulationGuid, consentSignature);
 
-        return Completable.fromAction(() -> dao.put(subpopulationGuid, consentSignature));
+        return consentSignature;
     }
 
-    /**
-     * Sends a copy of the participant's signed consent to their email address.
-     *
-     * @param subpopulationGuid guid for the subpopulation of the consent
-     * @return completable
-     */
-    @NonNull
```

Comment:
I'd love to, but may have to wait until I get back unless someone else does it. The code change should be trivial at this point, but I don't think I'll have time to test it sufficiently